release/ml_user_tests/ray-lightning/ray_lightning_user_test.py

```diff
@@ -9,11 +9,18 @@
     start = time.time()
 
     addr = os.environ.get("RAY_ADDRESS")
-    job_name = os.environ.get("RAY_JOB_NAME", "horovod_user_test")
+    job_name = os.environ.get("RAY_JOB_NAME", "ray_lightning_user_test")
+
+    # Manually set NCCL_SOCKET_IFNAME to "ens3" so NCCL training works on
+    # anyscale_default_cloud.
```

Comment:
Can you extend this comment with the reasoning why `ens3` works, or a link to a Github issue that explains the context?

Reply:
Done!